ext/opcache/ZendAccelerator.c

```diff
@@ -2746,9 +2746,9 @@ static void accel_move_code_to_huge_pages(void)
 	int mib[4] = {CTL_KERN, KERN_PROC, KERN_PROC_VMMAP, getpid()};
 	long unsigned int huge_page_size = 2 * 1024 * 1024;
 	if(sysctl(mib, 4, NULL, &s, NULL, 0) == 0) {
+		s = s * 4 / 3;
 		void *addr = mmap(NULL, s * sizeof (struct kinfo_vmentry), PROT_READ | PROT_WRITE, MAP_SHARED | MAP_ANON, -1, 0);
```

Comment:
https://github.com/radare/radare2/blob/master/libr/io/p/io_self.c#L572 calls malloc(size), but you mmap(..., s * sizeof (struct kinfo_vmentry), ...) and never free its result.

Reply:
2 * shame on me I wrote this radare's code ...